contributing/content-markup-reference.md

```diff
@@ -93,12 +95,15 @@ These instructions are pertinent to Windows users.
 
 {% endlinux %}
 ```
-```
-{% all %}
 
-Though rare, these instructions are pertinent to any other operating system.
+The platform-specific content matching the reader's operating system shows by default. You can override this behavior for individual Markdown files with the `defaultPlatform` frontmatter (such as for pages about GitHub Actions runners, which mostly use Linux):
 
-{% endall %}
+```
+---
+title: …
+versions: …
+defaultPlatform: linux
+---
```

Comment:
@Simran-B Let's document this new frontmatter value in our [docs on frontmatter](https://github.com/github/docs/tree/main/content#frontmatter) and then link to it in like 99 rather than sharing the example here. That way, all frontmatter values stay together in one place. ⚡ 

```suggestion
```